js/src/appBase.js

```diff
@@ -287,14 +333,28 @@ export class OasisAppBase {
       .then(processGetAddrEd25519Response, processErrorResponse);
   }
 
+  async getAddressAndPubKey_secp256k1(path) {
+    const data = await this.serializePath(path);
+    return this.transport
+      .send(this.CLA(), INS.GET_ADDR_SECP256K1, P1_VALUES.ONLY_RETRIEVE, 0, data, [0x9000])
+      .then(processGetAddrSecp256k1Response, processErrorResponse);
+  }
+
   async showAddressAndPubKey(path) {
```

Comment:
Can we append to **getAddressAndPubKey** and **showAddressAndPubKey** this prefix **_ed25519**